src/ucp/wireup/wireup_cm.c

```diff
@@ -251,24 +251,56 @@ ucp_cm_client_connect_prog_arg_free(ucp_cm_client_connect_progress_arg_t *arg)
     ucs_free(arg);
 }
 
-static void ucp_cm_client_restore_ep(ucp_wireup_ep_t *wireup_cm_ep,
-                                     ucp_ep_h ucp_ep)
+static void ucp_cm_copy_ep_lanes(ucp_ep_h to_ep, ucp_ep_h from_ep,
```

Comment:
done

Reply:
so in this case here should be 2 iterators and `to_ep` should be reconfigured since `to_ep` can have some lanes initialized.

Reply:
we have assert that it doesn't have lanes
to_ep and from_ep must have the same configuration

Reply:
ok

Reply:
which is removed?

Reply:
> which is removed?

`assert()` doesn't work, since configurations are not equal:
- TMP EP has two lanes: CM lane and transport lane
- UCP EP has only one lane: CM lane

After we copy UCT EPs from TMP EP to UCP EP, it will do `init_lanes()` where new config will be selected
The new config could be equal to what we have in TMP EP, then UCT EPs copied to UCP EP will be re-used in the new config. In the current code, it has to be the same.

Reply:
good catch, fixed